gravitee-am-ui/src/app/domain/settings/saml2/saml2.component.ts

```diff
@@ -55,11 +59,11 @@ export class Saml2Component implements OnInit {
       this.publicKeys(this.domainSamlSettings.certificate);
     }
 
-    this.pluginMetadata = {'deployed': false, 'feature': 'am-idp-saml2'};
     const samlPlugin = this.route.snapshot.data['identities']['am-idp-saml'];
```

Comment:
I think this part is wrong as this plugin is not the gateway idp but the idp saml